modules/magnolia/src/test/scala/pureconfig/module/magnolia/EnumerationsSuite.scala

```diff
@@ -0,0 +1,58 @@
+package pureconfig.module.magnolia
+
+import scala.language.higherKinds
+
+import com.typesafe.config.{ConfigFactory, ConfigValueFactory, ConfigValueType}
+import pureconfig._
+import pureconfig.error.WrongType
+import pureconfig.generic.error.NoValidCoproductOptionFound
+import pureconfig.module.magnolia.semiauto.reader._
+import pureconfig.module.magnolia.semiauto.writer._
+
+class EnumerationsSuite extends BaseSuite {
+
+  sealed trait Color
+  case object RainyBlue extends Color
+  case object SunnyYellow extends Color
+
+  val conf = ConfigFactory.parseString("{ type: person, name: John, surname: Doe }")
```

Comment:
We don't seem to be using this anywhere.

Reply:
It's being used in the first example of the spec. I moved its definition to an inner scope to make its use more evident in d805ae7.